docker-compose.yml

```diff
@@ -79,7 +79,7 @@ services:
       BAGCLI_DATABASE_PORT: 3306
       
   postgres:
-    image: postgres:11
+    image: postgres:latest
```

Comment:
Why not using a matrix to test with multiple Postgres versions ? Probably the same for other databases.